app-modules/task/tests/RequestFactories/EditTaskRequestFactory.php

```diff
@@ -53,7 +53,6 @@ public function definition(): array
             'status' => fake()->randomElement(TaskStatus::cases())->value,
             'due' => now()->addWeek(),
             'assigned_to' => User::factory()->create()->id,
-            'concern_type' => $contact->getMorphClass(),
             'concern_id' => $contact->getKey(),
```

Comment:
Now we can change this to just have `concern_id` be the `Contact::factory()` let's do so now.